Processo/src/App.java

```diff
@@ -6,9 +6,11 @@ public static void main(String[] args) {
 		System.out.println();
 		String [] candidatos = {"FELIPE", "MARCIA", "JULIA", "PAULO", "AUGUSTO"};
 		for(String candidato: candidatos) {
-			entrandoEmContato(candidato);
+			entrandoEmContato(candidato);   
 		}
 	}
+
+	//review
 	static void entrandoEmContato(String candidato) {
 		int tentativasRealizadas = 1;
 		boolean continuarTentando = true;
```

Comment:
Melhore tal coisa no metodo entrando EmContato
```suggestion
		boolean continuarTentando = false;
```
altere para false